### PR TITLE
Vue msg grp item

### PR DESCRIPTION
**Pull request by soujay**
The PR contains the display of failed message groups, add/edit/delete a comment associated with a note

### CODE REVIEW

File: src/ServicePulse.Host/vue/src/composables/serviceMessageGroup.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import {
+  useDeleteFromServiceControl,
+  usePostToServiceControl,
+  useFetchFromServiceControl,
+} from "./serviceServiceControlUrls.js";
+
+export function useGetExceptionGroups() {
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/vue/src/composables/serviceMessageGroup.js:7]: I'm not sure thrilled by the naming convention. What about dropping the "use" parts?

E.g. `getExceptionGroups() {`

File: src/ServicePulse.Host/vue/src/components/failedmessages/FailedMessageGroupNoteDelete.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<script setup>
+    const emit = defineEmits(["delete", "cancel"]);
+
+const settings1 = defineProps({
```
[the remark this answers is not in the document]
Comment (soujay) [reply src/ServicePulse.Host/vue/src/components/failedmessages/FailedMessageGroupNoteDelete.vue:4]: Sorry that was part of testing. I missed switching it back

File: src/ServicePulse.Host/vue/src/composables/serviceMessageGroup.js (hunk 1 above)
Comment (soujay) [reply src/ServicePulse.Host/vue/src/composables/serviceMessageGroup.js:7]: me neither. I continued to follow the conventions on how it was used in other vue pages.

Comment (TravisNickels) [reply src/ServicePulse.Host/vue/src/composables/serviceMessageGroup.js:7]: In the [documentation](https://github.com/Particular/Platform/blob/main/documentation/service-pulse.md#best-practises) it even states that the best practice is to have exported function begin with "use"

Comment (WilliamBZA) [reply src/ServicePulse.Host/vue/src/composables/serviceMessageGroup.js:7]: Ok, let's leave it as is for now. It feels super weird to me though

**Review by WilliamBZA — APPROVED** [review]

**Review by TravisNickels — APPROVED** [review]

**Pull request merged**